### ISSUE TITLE
Radial gradient with transparency has a not fully transparent background

### ISSUE THREAD

**Issue by RazrFalcon** [jrmuizel/raqote#103]
```rust
use raqote::*;

fn main() {
    let mut dt = DrawTarget::new(200, 200);

    let mut pb = PathBuilder::new();
    pb.rect(20.0, 20.0, 160.0, 160.0);
    let path = pb.finish();

    let gradient = Source::new_radial_gradient(
        Gradient {
            stops: vec![
                GradientStop {
                    position: 0.5,
                    color: Color::new(255, 255, 255, 255),
                },
                GradientStop {
                    position: 1.0,
                    color: Color::new(0, 0, 0, 0),
                },
            ],
        },
        Point::new(100.0, 100.0),
        80.0,
        Spread::Pad,
    );
    dt.fill(&path, &gradient, &DrawOptions::new());

    dt.write_png("test.png");
}
```

This code will crate a circle where transparent pixels actually have a 1 of 255 transparency, not 0. And white color in the center is not 255, but 254.

Here is an example:

![e-feDiffuseLighting-002](https://user-images.githubusercontent.com/725494/71644535-a7e86100-2cd2-11ea-8d95-8a28f590a0d6.png)

**Comment by jrmuizel** [comment]
This looks like the same problem as #62

**Comment by jrmuizel** [comment]
Actually, this should be easier to fix than #62 because the stops aren't close together.

**Comment by jrmuizel** [comment]
Updating to sw-composite 0.7.3 should fix the outer transparency. The inner part is still not completely white though.

**Comment by jrmuizel** [comment]
And sw-composite 0.7.4 should fix the inner part which means this should be fixed in raqote 0.7.9

**Comment by RazrFalcon** [comment]
Yes, it's way better now. But it still has some strange artifacts. Here is a comparison of [https://github.com/RazrFalcon/resvg-test-suite/blob/master/svg/e-feDiffuseLighting-002.svg](e-feDiffuseLighting-002.svg) (radialGradient + feDiffuseLighting filter).

![e-feDiffuseLighting-002](https://user-images.githubusercontent.com/725494/73909455-5e80c800-48b5-11ea-83b3-7cf85f48caaf.png)

PS: don't mind Inkscape and Batik, they are too dark because of bad filters support.
PSS: no firefox, because they broke down the screenshot feature
